transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/main/resources/jinjava/typeMappings/replayer.j2

```diff
@@ -0,0 +1,20 @@
+{%- import "common/route.j2" as rscope -%}
+{%- import "typeMappings/preserveAll.j2" as preserve -%}
+{%- include "typeMappings/rewriteDocumentRequest.j2" -%}
+{%- include "typeMappings/rewriteBulkRequest.j2" -%}
+{%- include "typeMappings/rewriteCreateIndexRequest.j2" -%}
+
+
+{%- set source_and_mappings = {
+  'request': request,
+  'index_mappings': index_mappings,
+  'regex_index_mappings': regex_index_mappings}
+-%}
+{{- rscope.route(source_and_mappings, request.method + " " + request.URI, flags, 'preserve.make_keep_json',
```

Comment:
we can print out the full map when we start the template.
However, what's required of the map will be a the decision of the caller and it might even be deferred all the way until runtime.  The map needs to correspond to the template.  Both are parameters, so they can be anything.  That's also why things are being added in pieces - though the lambda sent into the Jinjava transformer should be picking up most of that load.